README.md

```diff
@@ -34,93 +34,126 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 Leaderboard <a name="about-project"></a>
+# 📖 [PHOTO GALLERY] <a name="about-project"></a>
 
-The leaderboard website displays scores submitted by different players. It also allows you to submit your score
+Photo Gallery is a simple app that shows the photographs requested from the API https://picsum.photos. This project is the final Capstone Project of The JavaScript module from Microverse program's curriculum and has been done as a group Project by Andrea Rivas and Hien Phan.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
+<a href="https://www.w3.org/html/" target="_blank"><img align="center" src="https://raw.githubusercontent.com/devicons/devicon/master/icons/html5/html5-original-wordmark.svg" alt="html5" width="55" height="55"/></a><a href="https://www.w3schools.com/css/" target="_blank"><img align="center" src="https://raw.githubusercontent.com/devicons/devicon/master/icons/css3/css3-original-wordmark.svg" alt="css3" width="55" height="55"/></a><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript" target="_blank" rel="noreferrer"><img align="center" src="https://raw.githubusercontent.com/devicons/devicon/master/icons/javascript/javascript-original.svg" alt="javascript" width="55" height="55"/></a>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- [ ] **[Receiving photographs from API https://picsum.photos and render them in our app]**
+- [ ] **[Pop up window when you press button "Comments" ]**
+- [ ] **[Pop up window that renders all the info of the photography and the comments and a form to add your comment!]**
+- [ ] **[Sending the comments to the API: https://us-central1-involvement-api.cloudfunctions.net/ to be able to display them in the modal]**
+- [ ] **[Add likes when you press the heart like button]**
+- [ ] **[Likes are sent to the API https://us-central1-involvement-api.cloudfunctions.net/, and then we receive them in our counter likes]**
+- [ ]  **[Counter of the items displayed at the top of the home page]**
+- [ ]   **[Counter of items function tested with jest]**
+- [ ]    **[Counter of the comments in each modal]**
+- [ ]    **[Counter of comments function tested with jest]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🚀 Live Demo <a name="live-demo"></a>
-To see the demo version, you can <a href="https://hienphan0111.github.io/leaderboard/dist"> Click here </a>
+
+To see the demo version, you can a https://hienphan0111.github.io/JS-capstone-project/ 
+
+##  Video Documentation <a name="live-demo"></a>
+
+Here we explain you our project!
+
+https://drive.google.com/file/d/1M_C1MTr86qxF2wS3QZc8wYBOwot-4FcU/view?usp=sharing
```

Comment:
- [ ] **[OPTIONAL]** As you provided a presentation video in your README, it would be better to use a shareable link to your video so that the user is redirected  directly to the video without requesting access 🙏,

![image](https://user-images.githubusercontent.com/102757126/218065940-3b102e64-400b-4706-a955-477aefe9e930.png)

Reply:
Will do! Thank you for your review